pkg/kubectl/cmd/cmd_test.go

```diff
@@ -201,11 +201,17 @@ func NewAPIFactory() (*cmdutil.Factory, *testFactory, runtime.Codec) {
 		Validator: validation.NullSchema{},
 	}
 	generators := map[string]kubectl.Generator{
```

Comment:
I would have expected this to be a superset of the usually registered generators.